File: 1026java/Hw33.java
import java.util.Random;
import java.util.Scanner;

public class Hw33 {
    public static void main(String[] args) {
        Scanner Scan = new Scanner(System.in);
        System.out.print("What is the question ? \n");
            Scan.next();
            Pseudo();
    }
    public static void Pseudo(){
        String[] answer = {"Yes","No","Maybe","Ask me later"};
        Random Guess = new Random();
        int number = Guess.nextInt(4);
        System.out.print(answer[number]);
    }
}

File: 1027c/Hw33.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
	char question[200] = {'\0'}; int answer;
	printf("Ask me a question.\n");
	fgets(question,sizeof(question),stdin);
	srand(time(NULL));
	answer = 0;
	answer = (rand() % 4) + 1;
	switch (answer) {
	case 1: printf("Maybe"); break; system("pause");
	case 2: printf("Ask me later"); break; system("pause");
	case 3: printf("Yes"); break; system("pause");
	case 4: printf("No"); break; system("pause");
	}
	
	system("pause");
}